src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -41,7 +41,7 @@ Map<String, String> asProperties() {
       props.put(OS_ARCH_KEY, osArch);
       props.put(OS_NAME_KEY, osName);
       props.put(OS_VERSION_KEY, osVersion);
-      props.put(SIDECAR_VERSION_KEY, sidecarVersion);
+      props.put(VERSION, sidecarVersion);
```

Comment:
I think we should keep `SIDECAR_VERSION_KEY` here.

Reply:
yeah makes sense it's helpful to separate from all the other versions in the file